### PR TITLE
Adding description to integrations

### PR DESCRIPTION
**Pull request by l0k0ms**

### CODE REVIEW

File: content/integrations/airbrake.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ kind: integration
 doclevel: basic
 newhlevel: true
 git_integration_title: airbrake
+description: "View, search on, and discuss Airbrake exceptions in your event stream."
```
Comment (irabinovitch) [inline content/integrations/airbrake.md]: on is superfluous here.

File: content/integrations/awsapigateway.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ integration_title: AWS API Gateway
 kind: integration
 git_integration_title: amazon_api_gateway
 newhlevel: true
+description: "Track gateway errors, cache hits and misses, and request latency."
```
Comment (irabinovitch) [inline content/integrations/awsapigateway.md]: "Track Amazon API gateway errors"

File: content/integrations/awsautoscaling.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ integration_title: AWS Auto Scaling
 kind: integration
 git_integration_title: amazon_auto_scaling
 newhlevel: true
+description: "Track the status and counts of instances in your Auto Scaling groups."
```
Comment (irabinovitch) [inline content/integrations/awsautoscaling.md]: "Auto Scaling Groups "

File: content/integrations/awsbeanstalk.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ integration_title: AWS Elastic Beanstalk
 kind: integration
 git_integration_title: amazon_elasticbeanstalk
 newhlevel: true
+description: "Track key Amazon ElasticBeanstalk metrics."
```
Comment (irabinovitch) [inline content/integrations/awsbeanstalk.md]: AWS Elastic Beanstalk

File: content/integrations/awscloudtrail.md [hunk 5]
```diff
@@ -3,6 +3,7 @@ title: Datadog-AWS CloudTrail Integration
 integration_title: AWS CloudTrail
 kind: integration
 newhlevel: true
+description: "Alert on suspicious AWS account activity."
```
Comment (irabinovitch) [inline content/integrations/awscloudtrail.md]: This doesn't seem like the right description.

File: content/integrations/awscodedeploy.md [hunk 6]
```diff
@@ -4,6 +4,7 @@ integration_title: AWS CodeDeploy
 kind: integration
 git_integration_title: amazon_codedeploy
 newhlevel: true
+description: "See deployments as they happen and track how long they take."
```
Comment (irabinovitch) [inline content/integrations/awscodedeploy.md]: "See AWS CodeDeploy deployments  in your event stream as they happen,  as well as track completion times and success rates."

File: content/integrations/awsdynamo.md [hunk 7]
```diff
@@ -4,6 +4,7 @@ integration_title: AWS DynamoDB
 kind: integration
 git_integration_title: amazon_dynamodb
 newhlevel: true
+description: "Track table size, read/write capacity, request latency, and more."
```
Comment (irabinovitch) [inline content/integrations/awsdynamo.md]: "Track DynamoDB table size, read/write capacity, request latency, and more."

File: content/integrations/awsopsworks.md [hunk 8]
```diff
@@ -4,6 +4,7 @@ integration_title: AWS OpsWorks
 kind: integration
 git_integration_title: amazon_ops_works
 newhlevel: true
+description: "Track AWS OpsWorks resource usage."
```
Comment (irabinovitch) [inline content/integrations/awsopsworks.md]: Dont think this integration tracks resource usage. I think it tracks successes/failures and relevant run time metrics.

File: content/integrations/awspolly.md [hunk 9]
```diff
@@ -4,6 +4,7 @@ integration_title: AWS Polly
 kind: integration
 git_integration_title: amazon_polly
 newhlevel: true
+description: "Track key AWS Polly metrics."
```
Comment (irabinovitch) [inline content/integrations/awspolly.md]: Can we expand on this? What kind of metrics?

File: content/integrations/awsrds.md [hunk 10]
```diff
@@ -4,6 +4,7 @@ integration_title: AWS RDS
 kind: integration
 git_integration_title: amazon_rds
 newhlevel: true
+description: "Track a wealth of metrics related to Amazon RDS."
```
Comment (irabinovitch) [inline content/integrations/awsrds.md]: can we expand on this?

File: content/integrations/catchpoint.md [hunk 11]
```diff
@@ -3,6 +3,7 @@ title: Datadog-Catchpoint Integration
 integration_title: Catchpoint
 kind: integration
 doclevel: basic
+description: "Send your Catchpoint alerts to your Datadog event stream."
```
Comment (irabinovitch) [inline content/integrations/catchpoint.md]: can we expand on this?  What would you use catchpoint for? Why are those events useful?

File: content/integrations/chef.md [hunk 12]
```diff
@@ -5,6 +5,7 @@ kind: integration
 git_integration_title: chef
 newhlevel: true
 updated_for_agent: 5.8.5
+description: "Track Chef client runs: know when they fail, succeed, or make big changes."
```
Comment (irabinovitch) [inline content/integrations/chef.md]: Track Chef client runs including metrics on completion times, analytics on resource changes, and success rates.

File: content/integrations/cloud_foundry.md [hunk 13]
```diff
@@ -4,6 +4,7 @@ integration_title: Cloud Foundry
 kind: integration
 git_integration_title: cloud_foundry
 newhlevel: true
+description: "Track the health of your Cloud Foundry VMs and the jobs they run."
```
Comment (irabinovitch) [inline content/integrations/cloud_foundry.md]: We dont monitor job failures or VMs.

File: content/integrations/cloudhealth.md [hunk 14]
```diff
@@ -4,6 +4,7 @@ integration_title: Cloudhealth
 kind: integration
 git_integration_title: cloudhealth
 newhlevel: true
+description: "Help CloudHealth help you: give it per-instance metrics from Datadog."
```
Comment (irabinovitch) [inline content/integrations/cloudhealth.md]: Enrich and improve CloudHealth's right sizing and reservation recommendations with Datadog metrics.

File: content/integrations/desk.md [hunk 15]
```diff
@@ -3,6 +3,7 @@ title: Datadog-Desk Integration
 integration_title: Desk
 kind: integration
 git_integration_title: desk
+description: "See and discuss new, open, pending, and resolved cases in your event stream."
```
Comment (irabinovitch) [inline content/integrations/desk.md]: Should "Desk.com" be mentioned in the description? Isn't this used by Google for indexing?

File: content/integrations/dyn.md [hunk 16]
```diff
@@ -4,6 +4,7 @@ integration_title: Dyn
 kind: integration
 doclevel: basic
 git_integration_title: dyn
+description: "Monitor zone changes and track queries per second by zone or record."
```
Comment (irabinovitch) [inline content/integrations/dyn.md]: No mention of Dyn in the description.

File: content/integrations/ecs.md [hunk 17]
```diff
@@ -3,6 +3,7 @@ title: Datadog-AWS ECS Integration
 integration_title: AWS ECS
 kind: integration
 git_integration_title: amazon_ecs
+description: "Monitor container statuses, track resource usage, and more."
```
Comment (irabinovitch) [inline content/integrations/ecs.md]: AWS ECS integration doesn't monitoring individual containers but rather ECS clusters and tasks.

File: content/integrations/expressjs.md [hunk 18]
```diff
@@ -4,6 +4,7 @@ integration_title: "ExpressJS"
 kind: integration
 doclevel: basic
 git_integration_title: express
+description: "Monitor response times overall and request rates by response code."
```
Comment (irabinovitch) [inline content/integrations/expressjs.md]: no mention of Express.

File: content/integrations/google_cloud_platform.md [hunk 19]
```diff
@@ -3,6 +3,7 @@ title: Datadog-Google Cloud Platform Integration
 integration_title: Google Cloud Platform
 kind: integration
 git_integration_title: google_cloud_platform
+description: "Collect a wealth of GCP metrics and visualize your instances in a host map."
```
Comment (irabinovitch) [inline content/integrations/google_cloud_platform.md]: I'm not sure "wealth of" is appropriate here. Either we should describe the types of available metrics or explain why it is useful to have them.  GCP should probably be spelled out.

File: content/integrations/google_cloud_spanner.md [hunk 20]
```diff
@@ -5,6 +5,7 @@ kind: integration
 doclevel: basic
 newhlevel: true
 git_integration_title: google_cloud_spanner
+description: "Track the resource usage of your Spanner instances."
```
Comment (irabinovitch) [inline content/integrations/google_cloud_spanner.md]: Is instances the right term here?

File: content/integrations/google_container_engine.md [hunk 21]
```diff
@@ -5,6 +5,7 @@ kind: integration
 doclevel: basic
 newhlevel: true
 git_integration_title: google_container_engine
+description: "Monitor your GCE containers' resource usage."
```
Comment (irabinovitch) [inline content/integrations/google_container_engine.md]: Google Container Engine and/or GKE.  Not "GKE Containers"

File: content/integrations/honeybadger.md [hunk 22]
```diff
@@ -5,6 +5,7 @@ kind: integration
 doclevel: basic
 newhlevel: true
 git_integration_title: honey badger
+description: "View, search on, and discuss exceptions from Honeybadger in your event stream."
```
Comment (irabinovitch) [inline content/integrations/honeybadger.md]: "search on" -> "search"

File: content/integrations/puppet.md [hunk 23]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Datadog-Puppet Integration
 integration_title: Puppet
-
 doclevel: basicuppet
 kind: integration
+description: "Track Puppet runs: know when they fail, succeed, or make big changes."
```
Comment (irabinovitch) [inline content/integrations/puppet.md]: Big changes isn't the right wording here.

File: content/integrations/pusher.md [hunk 24]
```diff
@@ -4,6 +4,7 @@ integration_title: Pusher
 kind: integration
 git_integration_title: pusher
 newhlevel: true
+description: "Get metrics from Pusher into Datadog to see and monitor app engagement."
```
Comment (irabinovitch) [inline content/integrations/pusher.md]: reword please.

Metrics on what?

File: content/integrations/redmine.md [hunk 25]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Datadog-Redmine Integration
 integration_title: Redmine
-
 kind: integration
 doclevel: basic
+description: "View, search on, and discuss Redmine updates in your Datadog event stream."
```
Comment (irabinovitch) [inline content/integrations/redmine.md]: s/search on/search/

File: content/integrations/rollbar.md [hunk 26]
```diff
@@ -3,6 +3,7 @@ title: Datadog-Rollbar Integration
 integration_title: Rollbar
 kind: integration
 doclevel: basic
+description: "Send exceptions, errors, and code deployments to your Datadog event stream."
```
Comment (irabinovitch) [inline content/integrations/rollbar.md]: Send exceptions, errors, and code deployments from Rollbar to your Datadog event stream.

File: content/integrations/sentry.md [hunk 27]
```diff
@@ -3,6 +3,7 @@ title: Datadog-Sentry Integration
 integration_title: Sentry
 kind: integration
 doclevel: basic
+description: "See, search on, and discuss exceptions from Sentry in your event stream."
```
Comment (irabinovitch) [inline content/integrations/sentry.md]: s/search on/search/

File: content/integrations/servicenow.md [hunk 28]
```diff
@@ -2,6 +2,7 @@
 title: Datadog-ServiceNow Integration
 integration_title: ServiceNow
 kind: integration
+description: "Have your Datadog alerts automatically generate and update tickets."
```
Comment (irabinovitch) [inline content/integrations/servicenow.md]: Have your Datadog alerts automatically generate and update tickets in ServiceNow.

File: content/integrations/slack.md [hunk 29]
```diff
@@ -5,6 +5,7 @@ git_integration_title: slack
 kind: integration
 newhlevel: true
 doclevel: basic
+description: "Send Datadog alerts and graphs to your team's Slack channel."
```
Comment (irabinovitch) [inline content/integrations/slack.md]: *channels

File: content/integrations/zendesk.md [hunk 30]
```diff
@@ -5,6 +5,7 @@ kind: integration
 doclevel: basic
 newhlevel: true
 git_integration_title: zendesk
+description: "Track new versus solved tickets; auto-generate tickets from Datadog monitors."
```
Comment (irabinovitch) [inline content/integrations/zendesk.md]: mention zendesk

File: content/integrations/airbrake.md (hunk 1 above)
Comment (l0k0ms) [reply content/integrations/airbrake.md]: 👍

File: content/integrations/awsapigateway.md (hunk 2 above)
Comment (l0k0ms) [reply content/integrations/awsapigateway.md]: 👍

File: content/integrations/awsautoscaling.md (hunk 3 above)
Comment (l0k0ms) [reply content/integrations/awsautoscaling.md]: 👍

File: content/integrations/awsbeanstalk.md (hunk 4 above)
Comment (l0k0ms) [reply content/integrations/awsbeanstalk.md]: 👍

File: content/integrations/awscloudtrail.md (hunk 5 above)
Comment (l0k0ms) [reply content/integrations/awscloudtrail.md]: This is the desc on the corp website: 
image

File: content/integrations/awscodedeploy.md (hunk 6 above)
Comment (l0k0ms) [reply content/integrations/awscodedeploy.md]: 👍

File: content/integrations/awsdynamo.md (hunk 7 above)
Comment (l0k0ms) [reply content/integrations/awsdynamo.md]: 👍

File: content/integrations/awsopsworks.md (hunk 8 above)
Comment (l0k0ms) [reply content/integrations/awsopsworks.md]: identical to corporate website

File: content/integrations/awspolly.md (hunk 9 above)
Comment (l0k0ms) [reply content/integrations/awspolly.md]: 🆗

File: content/integrations/chef.md (hunk 12 above)
Comment (l0k0ms) [reply content/integrations/chef.md]: 👍

File: content/integrations/cloud_foundry.md (hunk 13 above)
Comment (l0k0ms) [reply content/integrations/cloud_foundry.md]: 👍

File: content/integrations/cloudhealth.md (hunk 14 above)
Comment (l0k0ms) [reply content/integrations/cloudhealth.md]: 👍

File: content/integrations/desk.md (hunk 15 above)
Comment (l0k0ms) [reply content/integrations/desk.md]: will do it on enhancement

File: content/integrations/google_cloud_spanner.md (hunk 20 above)
Comment (l0k0ms) [reply content/integrations/google_cloud_spanner.md]: yes

File: content/integrations/google_container_engine.md (hunk 21 above)
Comment (l0k0ms) [reply content/integrations/google_container_engine.md]: 👍

File: content/integrations/honeybadger.md (hunk 22 above)
Comment (l0k0ms) [reply content/integrations/honeybadger.md]: 👍

File: content/integrations/rollbar.md (hunk 26 above)
Comment (l0k0ms) [reply content/integrations/rollbar.md]: 👍

File: content/integrations/slack.md (hunk 29 above)
Comment (l0k0ms) [reply content/integrations/slack.md]: 👍

File: content/integrations/servicenow.md (hunk 28 above)
Comment (l0k0ms) [reply content/integrations/servicenow.md]: 👍

File: content/integrations/sentry.md (hunk 27 above)
Comment (l0k0ms) [reply content/integrations/sentry.md]: 👍

**Pull request merged**